File: Broken Phone.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y;
        scanf("%d%d",&x,&y);
        if(x>y)
        printf("NEW PHONE\n");
        else if(x<y)
        printf("REPAIR\n");
        else
        printf("Any\n");
    }
}

File: Broken Phone.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            if(x<y)
            System.out.println("REPAIR");
            else if(x>y)
            System.out.println("NEW PHONE");
            else
            System.out.println("ANY");
        }
    }
}